src/components/LoanCards/Buttons/LendIncrementButton.vue

```diff
@@ -77,8 +81,17 @@ export default {
 			// IF we wanted to show this interface on loans with less than 25 remaining they would see the selector
 			const minAmount = parseFloat(this.amountLeft < 25 ? this.loan.minNoteSize : 25); // 25_hard_coded
 			// cap at 20 prices
-			return buildPriceArray(this.amountLeft, minAmount).slice(0, 20);
-		}
+			const priceArray = buildPriceArray(parseFloat(this.amountLeft), minAmount).slice(0, 20);
+			// eslint-disable-next-line
+			if (this.isCompleteLoanActive && !priceArray.includes(Number(this.amountLeft).toFixed())) {
+				priceArray.push(Number(this.amountLeft).toFixed());
+			}
+			return priceArray;
+		},
+		isCompleteLoanActive() {
+			// eslint-disable-next-line
+			return isLessThan25(this.amountLeft) || isBetween25And500(this.amountLeft);
```

Comment:
Does this mean loans over $500 are not considered active in this situation?

Reply:
If the amount left of the loan is over $500 is not considered to complete the loan, mainly b/c $500 is the maximum amount option in the dropdown

<img width="122" alt="Screen Shot 2022-11-15 at 16 17 55" src="https://user-images.githubusercontent.com/94026278/202037092-27b246c1-66f8-4d16-b566-a278910076c6.png">